### ISSUE TITLE
User login & register page makeover

### ISSUE THREAD

**Issue by monsij** [monsij/StudentPortal#87]
**Describe the solution you'd like**
The present login and register view is pretty old-fashioned. Craft a new page to give it a more impressive and professional look. 🥇 

**Additional context**
You may like to see below what may the new UI look like:

![desktop](https://user-images.githubusercontent.com/23289387/50217546-f7513d00-03ae-11e9-851c-799e1c816aed.PNG)

![mobile](https://user-images.githubusercontent.com/23289387/50217635-38495180-03af-11e9-972d-1da31ef92e02.PNG)

**Comment by monsij** [comment]
[GitMate.io](https://gitmate.io) thinks a possibly related issue is https://github.com/monsij/StudentPortal/issues/47 (Improved user profile page).

**Comment by nguptaa** [comment]
Can i work on this too ? @monsij

**Comment by monsij** [comment]
Yeah sure. But make a point you do it fast. Also if you aren't working on #56 , comment so

**Comment by monsij** [comment]
Assigned : @nguptaa

**Comment by nguptaa** [comment]
> Any updates @nguptaa ?

@monsij Working on it. was little busy so couldn't complete it earlier.

**Comment by monsij** [comment]
@nguptaa  Integrate using the files attached
[Login_v3.zip](https://github.com/monsij/StudentPortal/files/2704872/Login_v3.zip)

**Comment by monsij** [comment]
Open for assignment
